### DISCUSSION TITLE
Removal of i686 Builds for Anaconda Package

### DISCUSSION THREAD

**Discussion by aaradhak** [rhinstaller/anaconda#5882] [General]
Summary:
Maintainers are encouraged to stop building their packages for i686, particularly if supporting this architecture demands considerable time or resources. However, this decision should exclude packages still required by other i686 packages or those used in a "multilib" context (e.g., for running 32-bit applications on x86_64 systems). Importantly, dropping i686 support from a leaf package will no longer be treated as a breaking change.
See: https://fedoraproject.org/wiki/Changes/EncourageI686LeafRemoval

In our case, running the [leafdrop](https://pagure.io/leafdrop) script shows that rpm-ostree is a package required by anaconda-install-img-deps, which highlights the need for discussion before removing i686 builds for any related package.

```
aaradhak@fedora ~/coderepo/rpms/leafdrop (main)$ ./leafdrop rpm-ostree
rpm-ostree is Required by:
- anaconda-install-img-deps

```
We propose starting an internal discussion within the team to evaluate whether i686 builds for anaconda-install-img-deps and other related packages can be dropped.

Ref: https://github.com/coreos/fedora-coreos-tracker/issues/1716

**Comment by jkonecny12** [comment]
Sounds good to me :), thanks for the heads-up.

**Reply by jkonecny12** [reply]
PR to solve this: https://github.com/rhinstaller/anaconda/pull/5887

In general Anaconda won't need i686 support because that system is not supported in Fedora and we are installing Fedora. Let's drop this to unblock other people.

**Reply by jkonecny12** [reply]
PR merged to Rawhide. Closing this discussion.
